sql/core/src/test/scala/org/apache/spark/sql/execution/UnsafeRowSerializerSuite.scala

```diff
@@ -40,9 +44,15 @@ class ClosableByteArrayInputStream(buf: Array[Byte]) extends ByteArrayInputStrea
 class UnsafeRowSerializerSuite extends SparkFunSuite {
 
   private def toUnsafeRow(row: Row, schema: Array[DataType]): UnsafeRow = {
-    val internalRow = CatalystTypeConverters.convertToCatalyst(row).asInstanceOf[InternalRow]
+    val converter = unsafeRowConverter(schema)
+    converter(row)
+  }
+
+  private def unsafeRowConverter(schema: Array[DataType]): Row => UnsafeRow = {
```

Comment:
Actually `UnsafeProjection.create(schema)` will do the codegen stuff, and this causes long time if we have to generate the large mount of `UnsafeRow`s.